src/main/createWindow.js

```diff
@@ -21,7 +21,7 @@ const createWindow = (pathname, options = {}) => {
     },
     useContentSize: true,
     show: false,
-    frame: false,
+    frame: (process.platform === 'linux'),
```

Comment:
This PR is helpful to the Linux users, One more thing, I think the `(` and `)` are not needed.

Can you remove them, then I will merge this PR.